Sources/SwerverCore/Action/GetAction.swift

```diff
@@ -6,8 +6,8 @@ public class GetAction: HttpAction {
     
     public func execute() -> HttpResponse {
         return responseBuilder
-            .withStatusCode(statusCode: StatusCode.ok.rawValue)
-            .withContentType(contentType: ContentType.text.rawValue)
+            .withStatusCode(statusCode: .ok)
```

Comment:
Nitpick: This should be `with(statusCode: .ok)`

This can be done in a future commit/PR.

Reply:
noted in new trello card